production.Dockerfile

```diff
@@ -27,5 +27,14 @@ RUN pip install -r requirements.txt --no-cache-dir --compile \
 
 RUN SECRET_KEY='unsafe secret key for collectstatic only' DATABASE_URL='postgres:///' REDIS_URL='redis:///' python manage.py collectstatic --noinput
 
+RUN apt-get purge -y git && apt autoremove -y
```

Comment:
I suppose `curl` and `build-essentials` are not needed after build either?

Reply:
We already purged them, but I did move this to the same line for clarity